app/webpacker/components/RegistrationsV2/Register/CompetingStep.jsx

```diff
@@ -172,15 +170,21 @@ export default function CompetingStep({
   };
 
   const actionUpdateRegistration = () => {
-    dispatch(setMessage('competitions.registration_v2.update.being_updated', 'basic'));
-    updateRegistrationMutation({
-      user_id: registration.user_id,
-      competition_id: competitionInfo.id,
-      competing: {
-        comment: hasCommentChanged ? comment : undefined,
-        event_ids: hasEventsChanged ? selectedEvents : undefined,
-      },
-      guests,
+    confirm({
+      content: i18n.t('competitions.registration_v2.update.update_confirm'),
+    }).then(() => {
+      dispatch(setMessage('competitions.registration_v2.update.being_updated', 'basic'));
+      updateRegistrationMutation({
+        user_id: registration.user_id,
+        competition_id: competitionInfo.id,
+        competing: {
+          comment: hasCommentChanged ? comment : undefined,
+          event_ids: hasEventsChanged ? selectedEvents : undefined,
+        },
+        guests,
+      });
+    }).catch(() => {
+      nextStep();
     });
```

Comment:
Am I reading it correctly that you're still proceeding to the `nextStep` even if the user does NOT confirm (which is what the `catch` branch represents in this case)?

Reply:
this brings the user back to the RegistrationOverview page